SofaKernel/modules/SofaBaseTopology/GridTopology.h

```diff
@@ -195,6 +210,9 @@ using MeshTopology::getHexahedron;
     Data<bool> d_computeHexaList, d_computeQuadList, d_computeEdgeList, d_computePointList;
     /// Data bool to set option to compute texcoords
     Data<bool> d_createTexCoords;
+
+    /// Enum storing the actual dimension of this grid
+    Grid_dimension m_gridDim;
```

Comment:
First idea was to use it from other components, like a tetrahedronFEM check that the grid is 3D to avoid crash.
But while testing I saw that we directly do test on the topology buffers.
So right now it is not used excepts in the test @damienmarchal added. But that the idea.